wordpress-mobile/WordPress-Android
# Issue/6134 async integrate video optimization

Fixes #6134 - integrates video optimization into the upload progress instead of making it a blocking task.

To test:

* Enable video optimization in app settings
* Create a post and add a video to it
* Notice that the upload progress in the editor and in the post list includes the optimization progress
